podcasts/ManageDownloadsCoordinator.swift

```diff
@@ -12,18 +12,18 @@ class ManageDownloadsCoordinator {
         else {
             return false
         }
-        return percentage < 0.1
+        if let lastCheckDate = Settings.manageDownloadsLastCheckDate,
+           fabs(lastCheckDate.timeIntervalSince(Date.now)) < 14.days {
```

Comment:
Change the code here to 5 seconds to make it easier to test if the banner shows up again.